app/views/layouts/application.html.erb

```diff
@@ -53,6 +53,7 @@
         <%= javascript_include_tag "application" %>
       <% end %>
       <%= yield(:page_meta) %>
+      <%# TODO Remove the zoom restriction, this is pretty bad for a11y %>
       <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```

Comment:
@benhalpern Do you happen to know why this is necessary? I feel like we should be able to remove it, but I didn't want to recreate a bug that this is hiding.

Reply:
Yes, it was originally in place to hide a bug and also to make the app more "app-like" for the mobile-wrapped context.

But I'm not even 100% sure the bug is even still present (Basically I think the width of the page is over 100% and this is kind of hiding that issue).

I'd rather fix this than hold onto the constraint due to a bug. @rhymes actually has an open PR from a while back where we discuss this.

https://github.com/thepracticaldev/dev.to/pull/2150

I want to get this fixed. I'm not even sure the old bug even still exists and I'm happy to forget about the native mobile parity concern in favor of a11y.

Reply:
Yes, it was originally in place to hide a bug and also to make the app more "app-like" for the mobile-wrapped context.

But I'm not even 100% sure the bug is even still present (Basically I think the width of the page is over 100% and this is kind of hiding that issue).

I'd rather fix this than hold onto the constraint due to a bug. @rhymes actually has an open PR from a while back where we discuss this.

https://github.com/thepracticaldev/dev.to/pull/2150

I want to get this fixed. I'm not even sure the old bug even still exists and I'm happy to forget about the native mobile parity concern in favor of a11y.

Reply:
Cool, thanks @benhalpern. I'll make a separate PR for that issue after I catch up on #2150.

Reply:
+1 on fixing it, I think the argument was there was, maybe, an issue in the iOS native app